internal/bucket.go

```diff
@@ -26,6 +27,51 @@ type BucketsCreateParams struct {
 	ShardGroupDuration string
 }
 
+func (p *BucketsCreateParams) Flags() []cli.Flag {
+	return []cli.Flag{
+		&cli.StringFlag{
+			Name:        "name",
+			Usage:       "New bucket name",
+			Aliases:     []string{"n"},
+			EnvVars:     []string{"INFLUX_BUCKET_NAME"},
+			Destination: &p.Name,
```

Comment:
We bind the values to the params structs, so we don't have to call `ctx.String`, etc

Reply:
I liked having the `cli.Command` and `cli.Flag` definitions all under `cmd/` and not in `internal/`, could we refactor back to that?

:+1: to setting `Destination` instead of repeating the flag names in `ctx.String` etc. calls

Reply:
I don’t see a problem doing it that way - could just move it to a helper function that binds the flags to a pointer of the params struct. One function for each. 

Bear in mind that your `main` package is going to get quite large if you throw every registered command in there. Most complex CLI applications end up with packages for each command and even sometimes one for each sub-command.

Reply:
Agreed about `main` getting big, and moving towards more sub-packages. I had the same thought around `internal`.